pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -763,4 +763,12 @@
      * </pre>
      */
     ConsumerBuilder<T> negativeAckRedeliveryBackoff(NegativeAckRedeliveryBackoff negativeAckRedeliveryBackoff);
+
+    /**
+     * Start the consumer in a paused state. When enabled, the consumer does not immediately fetch messages when
+     * {@link #subscribe()} is called. Instead, the consumer waits to fetch messages until {@link Consumer#resume()} is called.
+     * <p/>
+     * See also {@link Consumer#pause()}.
+     */
+    ConsumerBuilder<T> paused(boolean paused);
```

Comment:
@zwalsh-toast - what do you think about renaming this method to `startPaused`? It would then align with the configuration class.

```suggestion
    ConsumerBuilder<T> startPaused(boolean startPaused);
```

Reply:
done